src/compressed_tensors/quantization/lifecycle/apply.py

```diff
@@ -276,6 +277,29 @@ def find_name_or_class_matches(
         return matches
 
 
+def find_compression_targets(
```

Comment:
Agree on the naming, this seems likes a general utility function that, given a module input, will resolve all nested modules that match it. Ideally we should reflect that name as well as keeping this in a general utility package

Reply:
This is a more general function, that just takes in targets list for example ["Linear"] and expands it to all Linear modules, while ignoring things in the ignore_list for example ["lm_head"]; this is not tied to sparsity or quantization, I have changed the name to `expand_targets` which more closely matches what the function does, The thought behind putting it in this file is because this file already contains such general functions like `find_name_or_class_matches`